dowhy/gcm/stats.py

```diff
@@ -161,3 +164,65 @@ def permute_features(feature_samples: np.ndarray,
             np.random.shuffle(feature_samples[:, feature])
 
     return feature_samples
+
+
+def estimate_f_test_p_value(X_training_a: np.ndarray,
+                            X_training_b: np.ndarray,
```

Comment:
since dataset B needs to have less features than dataset A, we can consider making the names more explicit. E.g., `X_training` , `X_baseline_training`

Reply:
This makes the description harder. Describing a model A vs model B is easier than saying something like "A model trained on some data set vs a model trained on a baseline data set". However, I added a comment to the docstring for ```X_training_b``` to indicate that it should have less features than ```X_training_a```.
Or do you have another idea how to better connect the description with the parameter names?

Reply:
Good point. I am not sure either. Let's go with your proposal.